src/components/buttons/mint.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+require('react-dom');
+window.React2 = require('react');
```

Comment:
Could you leave a comment explaining the need of doing this?

Reply:
Checking if two different versions of react have been installed for the app.js and the components. The same issue that I've linked to above.